### PR TITLE
MBS-12632: Fix cases where [removed] is displayed in the relationship editor dialog preview

### PR DESCRIPTION
**Pull request by mwiencek**
https://tickets.metabrainz.org/browse/MBS-12632

### CODE REVIEW

**Comment by mwiencek** [comment]
Oh, I guess this also fixes https://tickets.metabrainz.org/browse/MBS-12653

**Comment by reosarevok** [comment]
Tried this locally and I now get [unknown] *all the time* inside popups, even if a name has been passed, and even if I change the name by hand after that :)

**Comment by mwiencek** [comment]
Sorry I forgot to amend the last commit, I think it should be fine now.

File: root/static/scripts/relationship-editor/components/DialogPreview.js [hunk 1]
```diff
@@ -97,7 +98,14 @@ const DialogPreview = (React.memo<PropsT>(({
...
     <EntityLink
       allowNew
-      content={content}
+      content={
+        nonEmpty(content)
+          ? content
+          : (
+            isDatabaseRowId(entity.id)
+              ? '' // have EntityLink determine the content
+              : (nonEmpty(entity.name) ? entity.name : l('[unknown]')))
```
Comment (reosarevok) [inline root/static/scripts/relationship-editor/components/DialogPreview.js:107]: FWIW I feel this is a bit more clear
```suggestion
            isDatabaseRowId(entity.id) || nonEmpty(entity.name)
              ? '' // have EntityLink determine the content
              : l('[unknown]'))
```

**Review by reosarevok — APPROVED** [review]
Works fine, possible small change for clarity suggested

**Pull request merged**